### DISCUSSION TITLE
CI Image Hosting

### DISCUSSION THREAD

**Discussion by kwryankrattiger** [SENSEI-insitu/SENSEI#112] [Ideas]
Due to the recent events around DockerHub I was wondering if it might be a good idea to move the CI images to ghcr.io/SENSEI-insitu. We are building a new set of images, since it has been almost a year, and now seems like the time to make that decision. I am not strongly opinionated either way, but I figure the timing makes it worth the discussion.

Some advantages to this is GitHub actions and GHCR live next to each other, and it would consolidate all of the SENSEI tools under the GitHub organization rather than being in two places. The other advantage? is anyone who has maintainer access to the SENSEI-insitu org in GitHub could push an image, which is everyone who works on the project now I think, so pushing updated the images wouldn't require @burlen or @kwryankrattiger or adding another person/creating access tokens to the Docker hub organization.

Other than that, I don't have any strong reason to move the images. We don't have that many images and free accounts are back and after the backlash I don't think Docker hub is going to try to drop the free account option again.

**Comment by burlen** [comment]
I don't have a strong opinion, do what you think is best here
